packages/ldc/ldc-phobos.patch.beforehostbuild

```diff
@@ -1,62 +0,0 @@
-diff --git a/std/stdio.d b/std/stdio.d
-index 0c315026..8b1860d0 100644
---- a/std/stdio.d
-+++ b/runtime/phobos/std/stdio.d
-@@ -310,6 +310,45 @@ else version (GENERIC_IO)
-         void funlockfile(FILE*);
-     }
- 
-+    version(CRuntime_Bionic)
-+    {
-+        import core.stdc.wchar_ : mbstate_t;
-+        import core.sys.posix.sys.types : pthread_mutex_t;
-+
-+        extern(C) struct wchar_io_data
-+        {
-+            mbstate_t  wcio_mbstate_in;
-+            mbstate_t  wcio_mbstate_out;
-+            wchar_t[1] wcio_ungetwc_buf;
-+            size_t     wcio_ungetwc_inbuf;
-+            int        wcio_mode;
-+        }
-+
-+        extern(C) struct __sfileext
-+        {
-+            __sbuf          _ub;
-+            wchar_io_data   _wcio;
-+            pthread_mutex_t _lock;
-+        }
-+
-+        void bionic_lock(FILE* foo)
-+        {
-+            if( foo == stdout._p.handle || foo == stdin._p.handle || foo == stderr._p.handle)
-+            {
-+                auto ext = cast(__sfileext*) foo._ext._base;
-+                if (ext._lock.value == 0)
```

Comment:
Bionic removed this `value` variable name years ago, which I just updated the D bindings to reflect, so this compatibility patch for 5.0 can't be used anymore.